internal/kafka/command_link_describe_tasks.go

```diff
@@ -31,9 +31,19 @@ type serializedTaskErrorOut struct {
 	ErrorMessage string `serialized:"error_message"`
 }
 
+func (c *linkCommand) newTasksCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "tasks",
+		Short: "Manager a cluster links tasks.",
+	}
+	cmd.AddCommand(c.newDescribeTasksCommand())
+
+	return cmd
+}
+
 func (c *linkCommand) newDescribeTasksCommand() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:               "describe-tasks <link>",
+		Use:               "describe <link>",
```

Comment:
Looks like this function constructs a list. On second thought, `confluent kafka link task list` would be a better command name in that case

Reply:
Is that recommended even though we are logically keying against a single link? I could see an argument describe is right because we are getting the tasks for one specific link.

But if you think list is better I can go with that.

Reply:
Updated the code with that but let me know your thoughts on the above as well.

Reply:
In general (for all of our commands) `describe` always prints a table and `list` always prints a list!